.scalafmt.conf

```diff
@@ -0,0 +1,9 @@
+version = "3.0.0-RC4"
```

Comment:
Looks like a lot of defaults have changed, it's better to use `version=1.5.1`

Reply:
Are they objectively bad or it's just a personal taste?
I'll do the change

Reply:
They're fine, I just want to unify the formatting with telegramium-core.

Reply:
That makes sense, then can't we just bring the `telegramium-core` up to `scalafmt` 3.0,0? or there's something that take us down?

Reply:
Can you make a PR to https://github.com/apimorphism/iozhik then? I like this config, what do you think?

```
version="3.0.0-RC5"
align.preset=more
maxColumn = 120
align.openParenCallSite = true
align.openParenDefnSite = true
continuationIndent.defnSite = 2
assumeStandardLibraryStripMargin = true
newlines.afterCurlyLambda = preserve
danglingParentheses.preset = true
importSelectors = singleLine
```

Reply:
I did the change here but the master of `iozhik` is not compiling.
This is the error I got by `sbt compile`
```
/home/benkio/workspace/iozhik/javatest/target/scala-2.12/classes ...
[error] /home/benkio/workspace/iozhik/iozhtest/src/main/scala/iozhik/vk/Poll.scala:25:18: Poll is already defined as case class Poll
[error] final case class Poll(id: Int,
[error]                  ^
[error] one error found
```

Reply:
Yeah, just replace the config

Reply:
https://github.com/apimorphism/iozhik/pull/6

Reply:
Thank you, it's merged. I modified the config a little: https://github.com/apimorphism/iozhik/blob/c68ec86436e0b45088e0854a5ad8b9841f0c10d2/iozhtest/src/main/resources/.scalafmt.conf, you can copy it and we'll merge this PR as well.